website/docs/examples/file_reader_gps/README.md

```diff
@@ -0,0 +1,135 @@
+---
+id: file-reader-gps
+title: File Reader GPS Example
+---
+
+This [file_reader_gps_example] (Link to gps_event_reader/main.py) provides a utility for reading and
+parsing GPS messages from the event file. It can process two types of GPS messages: relposned and pvt.
+The user specifies the type of GPS message to parse, and the script reads the corresponding data,
+unpacks it, and prints it to the console.
+
+:::info
+If you haven't already cloned the `farm-ng-amiga` repository, do
+so [**here**](/docs/brain/brain-install.md#clone-the-repository).
+:::
+
+:::tip
+If you already have the log file you want to run with this
+example you can skip to [**set up**](#setup)
+:::
+
+## Download the log file
+
+Now you are going to download the log file that you will use in
+this example.
+[**Click here to download**](https://farm-ng-dev-auto-plot-mvp.s3.us-west-2.amazonaws.com/datasets/western-growers-2022-12-05/events_12052022115852.bin)
+
+## [Optional] Make a Data folder
+
+We are going to make a folder that will store all of our log
+files, including the one you just downloaded.
+
+```bash
+cd <to-your-base-directory>
+mkdir <data-file>
+cd <data-file>
+pwd # the output of this is your <path>
+
+# Now that the file is downloaded you will do the following
+
+cd ~ # navigate to home directory
+cd Downloads
+mv events_12052022115852.bin <path-to-where-data-file-is-above> # moving the data to to data-folder
+```
+
+ Now that you have your log file in the correct place, in your
+ terminal navigate to where the repository `farm-ng-amiga` is and
+ open Visual Studio Code.
```

Comment:
This seems excessive, especially for a GPS example. We can have this in the log and access data example if you want
```suggestion
```